<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Katana Web Browser (0.01ALPHA)</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>-1</y>
      <width>321</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QWidget" name="titleWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>55</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(104, 104, 104, 214), stop:1 rgba(35, 35, 35, 194));

border-bottom: 2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 166));
border-bottom-left-radius:0px;
border-bottom-right-radius:0px;</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>0</y>
          <width>250</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="titleLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="1" column="0" colspan="11">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background:none;
color:white;
border:none;</string>
           </property>
           <property name="text">
            <string>Project Katana ALPHA 0.1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="7">
          <widget class="QLineEdit" name="lineEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:10px;
border: 1px solid rgba(255, 255, 255, 171);

padding:2px;
background-color:rgb(255,255,255,160);</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="goBack">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:0px;
border-bottom:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 166));

border-top:2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

border-right:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 171));

border-left:2px solid qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

background:rgb(70,70,70,255);</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="goForward">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:0px;
border-bottom:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 166));

border-top:2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

border-right:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 171));

border-left:2px solid qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

background:rgb(70,70,70,255);</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="9">
          <widget class="QToolButton" name="settingsButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
border-radius:0px;
border-bottom:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 166));

border-top:2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

border-right:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 171));

border-left:2px solid qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

background:rgb(70,70,70,255);
}</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QToolButton" name="homeButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:0px;
border-bottom:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 166));

border-top:2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

border-right:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 171));

border-left:2px solid qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

background:rgb(70,70,70,255);</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QToolButton" name="pageButton">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:0px;
border-bottom:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 166));

border-top:2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

border-right:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 171));

border-left:2px solid qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

background:rgb(70,70,70,255);</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="iconSize">
            <size>
             <width>14</width>
             <height>14</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="6">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="8">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="10">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="4">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>2</width>
             <height>2</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QToolButton" name="closeButton">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>20</width>
          <height>17</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:0px;
color:white;
border:1px solid rgb(255,255,255,150);
border-top-left-radius:5px;
border-bottom-left-radius:5px;
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(52, 52, 52, 255), stop:0.995238 rgba(135, 135, 135, 255));</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QToolButton" name="minimizeButton">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>5</y>
          <width>20</width>
          <height>17</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:0px;
color:white;
border:1px solid rgb(255,255,255,150);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QToolButton" name="maximizeButton">
        <property name="geometry">
         <rect>
          <x>43</x>
          <y>5</y>
          <width>20</width>
          <height>17</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:0px;
color:white;
border:1px solid rgb(255,255,255,150);
border-top-right-radius:5px;
border-bottom-right-radius:5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {

background-color: qlineargradient(spread:pad, x1:0.961905, y1:0, x2:1, y2:1, stop:0 rgba(43, 43, 43, 255), stop:1 rgba(90, 90, 90, 255));

border-radius:0px;
}

QToolButton {
border:none;
background:none;
color: rgb(217, 217, 217)
}

QToolButton:checked, QToolButton:hover {
border-radius:10px;
/*border-bottom:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 166));*/

/*border-top:2px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));

border-right:2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 171));

border-left:2px solid qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 172));*/

border:1px solid rgb(255,255,255,120);

background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.727, stop:0 rgba(64, 64, 64, 255), stop:0.671429 rgba(73, 143, 154, 230), stop:1 rgba(50, 128, 141, 217));
}

</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>319</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:0px;
border-top: 2px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 166));</string>
       </property>
       <widget class="QWidget" name="page"/>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Tab"/>
    <addaction name="actionNew_Window"/>
    <addaction name="actionNew_Private_Surf_Window"/>
    <addaction name="separator"/>
    <addaction name="actionAccounts"/>
    <addaction name="actionApplication_Settings"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionPlugin_Manager"/>
    <addaction name="separator"/>
    <addaction name="actionHistory_2"/>
    <addaction name="actionBookmarks"/>
    <addaction name="actionDownloads"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Page_As"/>
    <addaction name="actionPrint_Page"/>
    <addaction name="actionPrint_Page_Setup"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New Tab</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionAnimated_Interface">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Animated Interface</string>
   </property>
  </action>
  <action name="actionNew_Private_Surf_Window">
   <property name="text">
    <string>New Private-Surf Window</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Personal Preferences</string>
   </property>
  </action>
  <action name="actionAccounts">
   <property name="text">
    <string>Accounts</string>
   </property>
  </action>
  <action name="actionApplication_Settings">
   <property name="text">
    <string>Application Settings</string>
   </property>
  </action>
  <action name="actionPlugin_Manager">
   <property name="text">
    <string>Plugin Manager</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="actionHistory_2">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionCookies">
   <property name="text">
    <string>Cookies</string>
   </property>
  </action>
  <action name="actionBookmarks">
   <property name="text">
    <string>Bookmarks</string>
   </property>
  </action>
  <action name="actionDownloads">
   <property name="text">
    <string>Downloads</string>
   </property>
  </action>
  <action name="actionSave_Page_As">
   <property name="text">
    <string>Save Page As..</string>
   </property>
  </action>
  <action name="actionPrint_Page">
   <property name="text">
    <string>Print Page</string>
   </property>
  </action>
  <action name="actionPrint_Page_Setup">
   <property name="text">
    <string>Print Page Setup</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
